dev/com.ibm.ws.jpa_22_fat/fat/src/com/ibm/ws/jpa/jpa22/JPA22Injection.java

```diff
@@ -24,7 +33,17 @@
 
     @BeforeClass
     public static void setUp() throws Exception {
-        ShrinkHelper.defaultApp(server1, APP_NAME, "jpa22injection.web", "jpa22injection.entity");
+        final String resPath = "test-applications/jpa22/" + APP_NAME + "/resources/";
+
+        WebArchive app = ShrinkWrap.create(WebArchive.class, APP_NAME + ".war");
+        app.addPackage("jpa22injection.web");
+        app.addPackage("jpa22injection.entity");
+        app.merge(ShrinkWrap.create(GenericArchive.class).as(ExplodedImporter.class).importDirectory(resPath).as(GenericArchive.class),
```

Comment:
For this you could use `ShrinkHelper.addDirectory(app, resPath)`

Reply:
Yep, that looks a lot cleaner.  Applied.